mypy/defaults.py

```diff
@@ -6,13 +6,18 @@
 PYTHON3_VERSION = (3, 6)  # type: Final
 PYTHON3_VERSION_MIN = (3, 4)  # type: Final
 CACHE_DIR = '.mypy_cache'  # type: Final
-CONFIG_FILE = 'mypy.ini'  # type: Final
+INI_CONFIG_FILE = 'mypy.ini'  # type: Final
 SHARED_CONFIG_FILES = ['setup.cfg', ]  # type: Final
 USER_CONFIG_FILES = ['~/.config/mypy/config', '~/.mypy.ini', ]  # type: Final
 if os.environ.get('XDG_CONFIG_HOME'):
     USER_CONFIG_FILES.insert(0, os.path.join(os.environ['XDG_CONFIG_HOME'], 'mypy/config'))
+TOML_CONFIG_FILE = 'pyproject.toml'
```

Comment:
You can add a `# type: Final` annotation here (I believe this will help mypyc optimize things more, and improve correctness of course :^) ).